src/grid.hpp

```diff
@@ -0,0 +1,25 @@
+#include <AMReX.H>
+#include <AMReX_Array.H>
+#include <AMReX_Array4.H>
+#include <AMReX_Box.H>
+#include <AMReX_REAL.H>
+
+enum class centering { cc=0, fc, ec };
```

Comment:
put the enums and the struct inside a `namespace quokka { .. }` block

Reply:
Done.